main.go

```diff
@@ -36,42 +49,55 @@ func ProcessFile(profile config.ProcessProfileConfig, in io.Reader) error {
 
 func main() {
 
+	config_paths := new(inputConfigFilePaths)
+
 	// Parse command line.
-	ptr_config_path := flag.String("config", "", "config file path")
+	flag.Var(config_paths, "f", "Config file path (can be specified multiple times)")
 	flag.Parse()
 
-	// If path not specified, load defaultprofile from home directory.
-	config_path := *ptr_config_path
-	if *ptr_config_path == "" {
+	config_root := config.ConfigFileRoot{}
+
+	// Merge all config files, if any specified.
+	if len(*config_paths) != 0 {
+		loaded_configs := make([]config.ConfigFileRoot, 0) // Placeholder for loaded configs.
+		for _, path := range *config_paths {
+			conf, err := config.LoadConfigFromFile(path) // Load config file.
+			if err != nil {
+				log.Printf("[!] Error (%s) while loading config file: %s The config file will be ignored.\n", err, path)
+			}
+			loaded_configs = append(loaded_configs, conf) // Append to loaded configs.
+		}
+		config_root = config.MergeConfigFiles(loaded_configs...) // Merge all loaded configs.
+	} else { // If path not specified, load defaultprofile from home directory.
 
 		log.Printf("[!] Using default config file.")
 
-		var err error
-		config_path, err = getProfileFromHomeDir("default", true)
+		config_path, err := getProfileFromHomeDir("default", true)
```

Comment:
:= 

🤩🤩🤩

Reply:
One should use ':=' iff declaring new variable.
That's a pitfall. 🕳️